packages/orchestrator/internal/sandbox/fc/client.go

```diff
@@ -178,7 +184,15 @@ func (c *apiClient) setBootSource(ctx context.Context, kernelArgs string, kernel
 
 func (c *apiClient) setRootfsDrive(ctx context.Context, rootfsPath string) error {
 	rootfs := "rootfs"
-	ioEngine := "Async"
+
+	ioEngine, err := c.featureFlags.StringFlag(
```

Comment:
i am thinking if it isn't safer to do it as a boolean flag rather than passing a string from launch darkly. Or we should do some validation. A typo would break the flow. Is it case sensitive?